dune/grid/cpgrid/OrientedEntityTable.hpp

```diff
@@ -162,6 +164,36 @@ namespace Dune
 		return super_t::operator==(other);
 	    }
 
+            /** @brief Prints the relation matrix corresponding to the table, sparse format.
+
+             Let the entities of codimensions f and t be given by
+             the sets \f$E^f = { e^f_i } \f$ and \f$E^t = { e^t_j }\f$.
+             A relation matrix R is defined by
+	     \f{eqnarray*}{
```

Comment:
Probably, but I do not know for sure. The doc in question is partially copied from the pre-existing non-sparse version.